test/OrleansTestKit.Tests/OrleansTestKit.Tests.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="FluentAssertions" Version="6.11.0" />
     <PackageReference Include="Microsoft.CodeCoverage" Version="17.6.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.0" />
-    <PackageReference Include="Microsoft.Orleans.Sdk" Version="7.2.0" />
+    <PackageReference Include="Microsoft.Orleans.Sdk" Version="7.*" />
```

Comment:
This should be explicitly pinned to the version we want to build test. Changing it to a `*` makes the test runs non-deterministic.